mcsema/BC/Function.cpp

```diff
@@ -428,6 +428,21 @@ static void LiftBlockIntoFunction(TranslationContext &ctx) {
   }
 }
 
+static void AllocStackVars(
+    llvm::BasicBlock *bb,
+    const NativeFunction *cfg_func){
+  auto func = bb->getParent();
+  llvm::IRBuilder<> ir(bb);
+  const std::list<NativeStackVariable *> &stack_vars = cfg_func->stack_vars;
+  for (auto s : stack_vars){
+    auto char_type = llvm::Type::getInt8Ty(func->getContext());
+    auto int_type = llvm::Type::getInt64Ty(func->getContext());
```

Comment:
Move this out of the loop, or just use `word_type` as the type of the alloca size.